src/components/common/collapse-menu.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent, useState } from 'react';
+import { Button, Menu, MenuItem } from '@material-ui/core';
+import { observer } from 'mobx-react-lite';
+
+interface menuItem {
```

Comment:
We should keep the types/interfaces names capital case so it doesn't conflict with variable names

Reply:
Yep, updated